### PR TITLE
Added agent option to work with proxy

### PR DESCRIPTION
**Pull request by minimice**
I made a change so that it works behind proxies :)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ class IntrinioRealtime extends EventEmitter {
       this._throw("Need a valid options parameter")
     }
 
+    if (!options.agent) {
```
Comment (civilframe) [inline index.js]: Can you make this optional, so that it's backwards-compatible?

**Comment by minimice** [comment]
Made the change!

**Comment by minimice** [comment]
Hi can you review the change?

**Comment by civilframe** [comment]
Thanks! ⭐️

**Comment by civilframe** [comment]
@minimice Version 2.1.0 is out, with proxy support. Thanks for contributing!

https://www.npmjs.com/package/intrinio-realtime

**Pull request merged**